ksp_plugin_adapter/flight_planner.cs

```diff
@@ -266,8 +280,7 @@ class FlightPlanner : SupervisedWindowRenderer {
     for (int i = 0; i < burn_editors_.Count; ++i) {
```

Comment:
Since we have computed `first_future_manoeuvre_` already, could we not get rid of the loop, and just do it for `i == first_future_manoeuvre_`?

Reply:
Done.